pkg/operation/botanist/component/dependencywatchdog/bootstrap.go

```diff
@@ -55,28 +58,28 @@ const (
 	volumeMountPath = "/etc/dependency-watchdog/config"
 	configFileName  = "dep-config.yaml"
 
-	// ManagedResourceDependencyWatchdogEndpoint is the name of the dependency-watchdog-endpoint managed resource.
-	ManagedResourceDependencyWatchdogEndpoint = "dependency-watchdog-endpoint"
+	// ManagedResourceDependencyWatchdogWeed is the name of the dependency-watchdog-endpoint managed resource.
+	ManagedResourceDependencyWatchdogWeed = name + "-weed"
 	// ManagedResourceDependencyWatchdogProbe is the name of the dependency-watchdog-probe managed resource.
-	ManagedResourceDependencyWatchdogProbe = "dependency-watchdog-probe"
+	ManagedResourceDependencyWatchdogProbe = name + "-probe"
 )
 
 // BootstrapperValues contains dependency-watchdog values.
 type BootstrapperValues struct {
 	Role Role
-	ValuesEndpoint
+	ValuesWeed
 	ValuesProbe
 	Image string
 }
 
-// ValuesEndpoint contains the service dependants of dependency-watchdog.
-type ValuesEndpoint struct {
-	ServiceDependants restarterapi.ServiceDependants
+// ValuesWeed contains the service dependants of dependency-watchdog.
+type ValuesWeed struct {
```

Comment:
I am just wondering do we still need these structs? Why can't we directly put prober and weeder configurations in BootstrapperValues like:
```go
type BootstrapperValues struct {
  Role role
  WeederConfig weederapi.Config
  ProberConfig  proberapi.Config
  Image string
}
```

Reply:
updated as per suggestion in bdbb8bb